packages/pool/contracts/StakeUtils.sol

```diff
@@ -6,6 +6,12 @@ import "./interfaces/IStakeUtils.sol";
 
 /// @title Contract that implements staking functionality
 abstract contract StakeUtils is TransferUtils, IStakeUtils {
+
+
+    string private constant NOT_ENOUGH_FUNDS = "API3DAO.StakeUtils: USER DONT HAVE ENOUGH TOKENS TO STAKE/UNSTAKE THE PROVIDED AMOUNT";
+    string private constant UNSTAKE_TIMING_ERROR = "API3DAO.StakeUtils: USER IS ALLOWED TO UNSTAKE NO LONGER THEN A WEEK FROM SCHEDULED TIME";
+    string private constant STAKING_ADDRESS_ERROR = "API3DAO.StakeUtils: IT IS ONLY POSSIBLE TO STAKE TO YOURSELF";
```

Comment:
pls move `ERROR` to the front for the consistency, so we can have something like `ERROR_UNSTAKE_TIMING`, `ERROR_NOT_ENOUGH_FUNDS`

Reply:
etc.

Reply:
Fixed